x11-misc/trayer/files/trayer-1.0-fix-incompatible-pointer-types.patch

```diff
@@ -0,0 +1,19 @@
+Clang16 will not allow incompabile pointer types by default.
+
+Bug: https://bugs.gentoo.org/881081
+
+Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>
+--- a/plugin.h
++++ b/plugin.h
+@@ -9,6 +9,8 @@
+ #include <stdio.h>
+ #include "panel.h"
+ 
++struct _plugin;
```

Comment:
This had me confused at first, but then I realised it was an implicit int before!

Reply:
Heh, yeah that was a tricky one